tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/Stylesheets.spec.ts

```diff
@@ -0,0 +1,184 @@
+﻿import {ConstantHelper, test} from '@umbraco/playwright-testhelpers';
+import {expect} from "@playwright/test";
+
+
+test.describe('Stylesheets tests', () => {
+
+  const styleSheetName = 'TestStyleSheetFile';
+  const styleSheetFileName = styleSheetName + ".css";
+  const ruleName = 'TestRuleName';
+  const styleFolderName = 'TestStylesheetFolder';
+  
+  test.beforeEach(async ({umbracoUi}) => {
+    await umbracoUi.goToBackOffice();
+    await umbracoUi.goToSection(ConstantHelper.sections.settings);
+  });
+  
+
+  test('can create a stylesheet file', async ({umbracoApi, umbracoUi}) => {
+    // Arrange
+    await umbracoApi.stylesheet.ensureNameNotExists(styleSheetFileName);
+
+    //Act
+    await umbracoUi.stylesheet.openActionsMenuAtRoot();
+    await umbracoUi.stylesheet.clickNewStylesheetFileButton();
+    await umbracoUi.stylesheet.enterStylesheetName(styleSheetName);
+    await umbracoUi.stylesheet.clickSaveButton();
+
+    // Assert
+    expect(await umbracoApi.stylesheet.doesExist(styleSheetFileName)).toBeTruthy;
+    // TODO: when frontend is ready, verify the new stylesheet is displayed under the Stylesheets section 
+    // TODO: when frontend is ready, verify the notification displays
+    
+    // Clean
+    await umbracoApi.stylesheet.ensureNameNotExists(styleSheetFileName);
+  });
+
+  test('can create a new Rich Text Editor style sheet file', async ({umbracoApi, umbracoUi}) => {
+    // Arrange
+    await umbracoApi.stylesheet.ensureNameNotExists(styleSheetFileName);
+
+    //Act
+    await umbracoUi.stylesheet.openActionsMenuAtRoot();
+    await umbracoUi.stylesheet.clickNewRTEStylesheetFileButton();
+    await umbracoUi.stylesheet.enterStylesheetName(styleSheetName);
+    await umbracoUi.stylesheet.addNewRule(ruleName, 'h1', 'color:red');
+    await umbracoUi.stylesheet.clickSaveButton();
```

Comment:
```suggestion
    await umbracoUi.waitForTimeout(1000);
    await umbracoUi.stylesheet.clickSaveButton();
```

Reply:
waitForTimeout is unnecessary here as the test still works fine without it